new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleValidation.java

```diff
@@ -61,27 +62,31 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+//import oracle.weblogic.kubernetes.extensions.LoggedTest;
+
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Simple validation of integration tests")
 // Every test class needs to tagged with this annotation for
 // diagnostic log collection in case of failures and for test cleanup.
 @IntegrationTest
-class ItSimpleValidation implements LoggedTest {
+class ItSimpleValidation {
 
   private String opNamespace = null;
   private String domainNamespace1 = null;
 
   final String domainUid = "domain1";
   String serviceAccountName;
 
+  private static LoggingFacade logger = null;
+
   /**
    * Setup for test suite. Creates service account, persistent volume
    * and persistent volume claim.
    * @param namespaces injected by JUnit extension
    */
   @BeforeAll
   public void setup(@Namespaces(2) List<String> namespaces) {
-
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.